R/get_sc_ch_episodes_path.R

```diff
@@ -0,0 +1,20 @@
+#' Function for reading in all care home episodes
```

Comment:
Simplify/correct the titles of functions 
```suggestion
#' Get the Care Home all episodes file path
```